local-volume/bootstrapper/README.md

```diff
@@ -1,9 +1,70 @@
 # local-volume-provisioner-bootstrap
 
-local-volume-provisioner-bootstrap is used to bootstrap provisioner. The main use
-case of bootstrapper is to make provisioner configurable. It configures provisioner
-based on user config, creates appropriate service account, role bindings, then starts
-provisioner. It will exit as soon as provisioner is successfully created.
+local-volume-provisioner-bootstrap is used to bootstrap provisioner. The main use case of bootstrapper is to make
+provisioner configurable. Below is a detailed flow of how the bootstrap process works:
+
+- looks for a configmap passed in via flag `-volume-config`; otherwise, a default configmap named `local-volume-default-config` will be created
+- reads and validates the configmap, then auto-generates missing configurations, see below
+- looks for a service account passed in via flag `-serviceaccount`; otherwise, a default service account named `local-storage-admin` will be created
+- creates two cluster role bindings for the service account, i.e. `system:persistent-volume-provisioner` and `system:node`.
```

Comment:
I think system:node actually has more privileges than we need.  Maybe as a future enhancement, we can look into using a different role with fewer permissions.

Also, are there any roles needed for the boostrapper to update/create the configmap?